ixofoundation/ixo-world
# chore: add links to android and ios mobile apps

## Scope
https://nonaredteam.atlassian.net/browse/IXO-613

## Work done
Link to extension based on browser
Content changes to footer

## Steps to test
Click on Mobile link in footer on Chrome & Safari
See updated footer:
T's & C's -> Terms
Removed dot between location and Terms
IXO -> ixo for About section
